### DISCUSSION TITLE
AI Deck Camera Images Brightness

### DISCUSSION THREAD

**Discussion by migranram** [bitcraze/discussions#180] [Q&A] [answered]
Hello everyone, I have been doing some tests with the AI Deck RGB Camera and there is something I cannot find a solution for in the code.
I have realized that the brightness of the images is only set up when the Crazyflie boots up. Therefore if I turn the drone on in a dark area and then go to a brighter one the images are fully over-saturated. 
Does anyone knows how to solve this and make the brightness to adjust, or if it is even possible with the current firmware? 

I have been looking in the code of the [wifi-streamer example](https://github.com/bitcraze/aideck-gap8-examples/tree/master/examples/other/wifi-img-streamer) and the bit in the firmware but I do not find anything there.

Thank you in advance!

**Comment by knmcguire** [comment, marked as answer]
Hi! Have you seen [this issue ](https://github.com/bitcraze/aideck-gap8-examples/issues/63) yet? Perhaps you can try what that user has tried: https://github.com/bitcraze/aideck-gap8-examples/issues/63

**Reply by migranram** [reply]
Yes I just tried, but it seems to get frozen, it only sends one picture and then nothing else.

**Reply by migranram** [reply]
Sorry, my bad! It seems to work now. Thank you!

**Reply by knmcguire** [reply]
ah great thanks for confirming!
